instrumentation/lettuce-4.3/src/main/java/com/nr/fit/lettuce/instrumentation/NRBiConsumer.java

```diff
@@ -0,0 +1,38 @@
+package com.nr.fit.lettuce.instrumentation;
```

Comment:
We shouldn't have any packages containing `fit` (aka Field Instrumentation Team) if we're porting this from their experimental repo to the agent proper.

Reply:
ok